surefire-its/src/test/resources/surefire-1614-stream-corruption/src/test/java/com/example/demo/CustomRunner.java

```diff
@@ -0,0 +1,19 @@
+package com.example.demo;
+
+import org.junit.runners.BlockJUnit4ClassRunner;
+import org.junit.runners.model.InitializationError;
+import org.junit.runners.model.TestClass;
+
+public class CustomRunner extends BlockJUnit4ClassRunner {
```

Comment:
Still not the ASF Maven style.
Why if you type it correctly this is another check style.

Reply:
So we are not running checkstyle on IT cases? Fair. But we can consider adding it

Reply:
I just copied the sample project files. Did not manually type a character here.

That'd be wonderful! As long as "mvn verify" exits with "0", I'm fine with the format.

Even better: have profile or task that applies the desired formatting automatically via spotless or other formatting tool. For all sources, be it `src/main`, `src/test`, or `src/it`.

Reply:
If you use IDEA14 ad it is configured properly, just press CTRL+ALT+L and the files are adjusted to ASF style check.

Reply:
All I'm saying is that when [an almost 2 hour running build](https://builds.apache.org/job/maven-box/job/maven-surefire/job/SUREFIRE-1614/) doesn't fail due to code formatting issues, everything's fine for me too.

> [...] use IDEA14 ad it is configured properly [...]

There are tools that a) format and b) check for style violations. We should use them -- and not rely on personal IDEs, their proper configuration and hope that stars align. If such tools were used you even gain speed when reviewing PRs: a successful build means that all style checks passed. Yeah!

> You have to follow the code style

Sure. Still I expect from properly setup projects to notify me about code style violations. A build should not even start to compile stuff when there are code style violations present.

> I guess you want to work in ASF

?